sdk/servicebus/service-bus/test/sessionsTests.spec.ts

```diff
@@ -269,7 +269,7 @@ describe("session tests", () => {
         await receiver.getSessionState({ abortSignal: controller.signal });
         throw new Error(`Test failure`);
       } catch (err) {
-        err.message.should.equal("The getState operation has been cancelled by the user.");
+        err.message.should.equal("The operation was aborted.");
```

Comment:
Should we instead check that the error is an instance of `AbortError`?

Reply:
Makes total sense to me.